Packages/vcs/Lib/vtk_ui/text.py

```diff
@@ -40,7 +149,7 @@ def text_actor(string, fgcolor, size, font):
 
     # Sane defaults.
     props.SetJustificationToCentered()
-    props.SetVerticalJustificationToCentered()
+    props.SetVerticalJustificationToTop()
```

Comment:
Made Top the default justification for my text– shouldn't impact VCS too much, as the justifications for that are handled before being passed into the VTK UI layer.